src/components/CurriculumComponents/CurriculumHeaderTabNav/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { ButtonVariant } from "@/components/SharedComponents/Button/common";
+import ButtonAsLink, {
+  ButtonAsLinkProps,
+} from "@/components/SharedComponents/Button/ButtonAsLink";
+import Flex, { FlexProps } from "@/components/SharedComponents/Flex.deprecated";
+
+/**
+ * CurriculumHeaderTabNav is a 'nav' component which renders a tab nav specific to curriculum pages.
+ *
+ * ## Usage
+ *
+ * It is only used in the CurriculumHeader component so far.
+ */
+const CurriculumHeaderTabNav = ({
+  label,
+  links,
+  variant = "flat",
+  ...flexProps
+}: FlexProps & {
+  label: string;
+  links: ButtonAsLinkProps[];
+  variant?: ButtonVariant;
+}) => {
+  return (
+    <Flex
+      as="nav"
+      aria-label={label}
+      $mv={0}
+      $pv={0}
+      $overflowY={"hidden"}
+      $overflowX={"auto"}
+      {...flexProps}
+    >
+      {links.map((link, i) => (
+        <ButtonAsLink
+          {...link}
+          size="large"
+          variant={variant}
+          aria-current={link.isCurrent ? "page" : undefined}
+          aria-controls={link.page}
+          key={`CurriculumHeaderTabNav-${link.page}-${i}`}
```

Comment:
@HelenHarlow @assadk88 If we implemented the extra line I've added below on this component, then the following would apply for the currently active tab:
* The link would no longer be clickable or activatable using a keyboard.
* The link would not have any hover or focus styles (currently it has focus styles but not hover styles)
* The link would no longer be in the tab index.
* The link would be reported as a `dimmed` or `disabled` link by screen readers.
* The link would still be in screen readers' list of links.

Is this a feature we want, or is it better that the link be clickable even when it is pointing to the currently active tab? We could also tackle this as a separate issue, as it's not an accessibility issue to leave it as-is.

```suggestion
          aria-controls={link.page}
          disabled={link.isCurrent ? true : undefined}`
          key={`CurriculumHeaderTabNav-${link.page}-${i}`}
```

Reply:
Hey! The reason it's missing hover styles currently is that we would need to add an additional hover state for the 'selected' tab, as the component looks different (it has a line on the top). This didn't feel necessary seeing as clicking on that tab in the selected state won't do anything but reload the page. As long as screen reader users can still obtain the full list of links in the tabs, as this is important for understanding the structure of the page, then I'm happy with whatever approach. Hopefully this extra context helps?

Reply:
Yup, makes sense, so my question to you is would you like the currently active tab to be a clickable link, or would you prefer it to be a 'disabled'/'dimmed' link? There is no a11y impact either way, so it's more of a UX/Design question.

Reply:
If we left it as is, the link still would be reported as 'current' or 'active' by screen readers, so even though it's clickable it should still be clear that activating the link won't do anything except refresh the page.

Reply:
Great, thanks!